doc/examples.xml

```diff
@@ -309,3 +309,36 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="TadpoleDigraph">
+<ManSection>
+  <Oper Name="TadpoleDigraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are non-negative integers (where <A>m</A>> 2), then this operation
+    returns a Tadpole digraph.<P/>
```

Comment:
In fact I don't think you should allow `n` to be `0` - I don't think it makes sense. In any case you would need to define what the tadpole digraph would look like with n = 0 (is there a bridge or not?) but that would be a bit arbitrary.